### DISCUSSION TITLE
How to Disable the action that makes it full screen on double click

### DISCUSSION THREAD

**Discussion by teppeiabe** [SuRGeoNix/Flyleaf#452] [Q&A]
By default, double-clicking the Player toggles full screen. Is there a way to disable this, and is it possible to assign a different event to the double-click action?

**Comment by RobAdams27** [comment]
I know this is an old question but in the FlyleafHost there is a property called ToggleFullScreenOnDoubleClick which you can set to "false" to disable double-clicking to maximize.
